<launch>
<node name="soundplay_node" pkg="sound_play" type="soundplay_node.py" output="screen"/>
  <node name="tts_node" pkg="tts" type="tts_node.py" output="screen">
    <rosparam command="load" file="$(find tts)/config/pegabot_params.yaml" />
  </node>
</launch>

